src/Type/Comment/Mutation/CommentCreate.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace WPGraphQL\Type\Comment\Mutation;
+
+use GraphQL\Error\UserError;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQLRelay\Relay;
+use WPGraphQL\AppContext;
+use WPGraphQL\Type\WPInputObjectType;
+use WPGraphQL\Types;
+/**
+ * Class CommentCreate
+ *
+ * @package WPGraphQL\Type\Comment\Mutation
+ */
+class CommentCreate { 
+
+	/**
+	 * Holds the mutation field definition
+	 *
+	 * @var array $mutation
+	 */
+	private static $mutation;
+	/**
+	 * Defines the create mutation for Comments
+	 *
+	 * @return array|mixed
+	 */
+	public static function mutate() {
+        if (empty(self::$mutation)) {
+			$mutation_name = 'CreateComment';
+			self::$mutation = Relay::mutationWithClientMutationId([
+				'name' => $mutation_name,
+				'description' => __('Create comment objects', 'wp-graphql'),
```

Comment:
Not a _huge deal_ but we _mostly_ try and follow WordPress core code standards in terms of spacing, etc. 

If you're using PHPStorm you can easily set the project to use the WordPress code style then reformat the files.

Reply:
I'm using VSCode but I can use Wordpress spacing from now on and make changes to my code. I kinda hate the spacing because is make reading it from a mobile device kinda unbearable.